app/email/index.js

```diff
@@ -27,4 +27,24 @@ const sendPasswordReset = (email, firstName, code) => {
   });
 };
 
-module.exports = { sendPasswordReset };
+const emailVerificationTemplate = readTemplate('emailVerification.ejs');
+const sendEmailVerification = async (email, firstName, code) => new Promise((resolve, reject) => {
+  const data = {
+    to: email,
+    from: config.email.user,
+    subject: 'ACM UCSD Membership Portal Email Verification',
+    html: ejs.render(emailVerificationTemplate, {
+      firstName,
+      link: `${config.client}/verifyEmail/${email}/${code}`,
+    }),
+  };
+  sendEmail(data).then(() => {
+    log.info(`Sent verification email to ${email}`);
+    resolve();
+  }).catch((error) => {
+    log.warn(`Failed to send email verification email to ${config.email.user}: ${error}`);
+    reject();
+  });
+});
```

Comment:
Instead of wrapping the whole thing in a Promise, can you return the `sendEmail` promise? And maybe throw an error in the `catch` block since the database transaction depends on it.